app/views/admin_users/rewards/show.html.erb

```diff
@@ -2,13 +2,13 @@
   <figure class="text-end">
     <figcaption class="block-footer">
       <div class="row">
-      <div class="col-6 text-start">
-        <strong>Category:</strong>
-        <%= reward.category? %>
+        <div class="col-6 text-start">
+          <strong>Category:</strong>
+          <%= reward.category? %>
```

Comment:
the `category?` method name is a bit misleading - looks like the enum is underneath - I would consider renaming it to `display_category` or something like that

Reply:
Thanks a lot for CR :) I changed this method name 👍